cases/templates/cases/case_list_staff.html

```diff
@@ -13,6 +13,10 @@ <h2 class="govuk-visually-hidden">Filters</h2>
         {% crispy filter.form %}
         </div>
     </div>
+    <script>
+        var nw = nw || {};
+        nw.user_wards = {{ user_wards|safe }};
```

Comment:
If the user has no wards, this `{{ user_wards|safe }}` evaluates to `None`, which causes a JavaScript "None is not defined" error.

Reply:
Stupid empty defaults, ta, will fix.